terraform/node_resource_plan_instance.go

```diff
@@ -107,8 +107,8 @@ func (n *NodePlannableResourceInstance) evalTreeManagedResource(addr addrs.AbsRe
 	var provider providers.Interface
 	var providerSchema *ProviderSchema
 	var change *plans.ResourceInstanceChange
-	var refreshState *states.ResourceInstanceObject
-	var planState *states.ResourceInstanceObject
+	var instanceRefreshState *states.ResourceInstanceObject
```

Comment:
I quite appreciate this clearer variable, as you know I've been knee-deep in this code (and lost and confused) recently.